### ISSUE TITLE
issues

### ISSUE THREAD

**Issue by linhus** [DanWin/hosting#46]
Hi there are are lot of issues in the repositorys, hm i try with a new setup and post the install setup here.

**Comment by linhus** [comment]
i use debian 9.8 with the latest updates

**Comment by DanWin** [comment]
Indeed, with Debian 9 there are some issues as systemd is too outdated to know the BindPaths option for example. You will have to downgrade the "ProtectSystem=strict" option to a "ProtectSystem=full" in all the systemd config files. There may also be some other issues you will encounter. My setup is running the latest version from the unstable channel "sid", so it might be that not everything will work out of the box on other distributions/versions.

**Comment by linhus** [comment]
yes the systemd was a big issue, so i try with a clean install with debian sid.

**Comment by linhus** [comment]
ok i have installed the latest version from the unstable channel, but i cannot start the default service php7.3-fpm@default.server

**Comment by linhus** [comment]
the next is the custom.conf from the nginx service,
when i start: failed at step NAMESPACE spawning /usr/sbin/ngix: no such file or directory

**Comment by linhus** [comment]
nginx.service - A high performance web server and a reverse proxy server
   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
  Drop-In: /etc/systemd/system/nginx.service.d
           └─custom.conf
   Active: failed (Result: exit-code) since Fri 2019-03-08 19:14:31 CET; 6min ago
     Docs: man:nginx(8)
  Process: 6312 ExecStartPre=/usr/bin/install -Z -m 02755 -o www-data -g www-data -d /run/nginx (code=exited, status=226/NAMESPACE)

**Comment by linhus** [comment]
i have deltet the nginx.service.d and its working :-)

**Comment by linhus** [comment]
Have you a idea with the:
Could not create initial preference file! /var/local/squirrelmail/data/ should be writable by the user www-data. Please contact your system administrator and report this error.

**Comment by linhus** [comment]
ok the only way was to move the folder to
/var/www/html/squirrelmail/data/
and it will work, but is not the best solution
